A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU.html

```diff
@@ -1,132 +1,132 @@
-﻿<link rel="stylesheet" href="A320_Neo_CDU.css" />
```

Comment:
I think your commit messed up the line endings here. You probably will want to revert this file.